### ISSUE TITLE
Pods networking is broken after openvswitch is restarted

### ISSUE THREAD

**Issue by bbl** [openshift/sdn#47]
#### Description

After `ovs` pod is restarted all pods on the corresponding node come up with broken networking. The gateway is not reachable, thus all egress connections are not possible.
If `ovs-ofctl -O OpenFlow13 dump-ports-desc br0` is run inside the `ovs` pod, the output doesn't show old vethXXX interfaces, however they're still present on host.

##### Version

* The output of `git describe` of openshift-ansible 
```
openshift-ansible-3.11.146-1-22-g37e13e5
```
* `ovs` image version:

```
docker.io/openshift/origin-node:v3.11
```

##### Steps To Reproduce
1. Delete/restart the `ovs` pod on the compute node. 
2. Run `ovs-ofctl -O OpenFlow13 dump-ports-desc br0`, verify that `veth` interfaces are missing. 

##### Expected Results
Expected pod networking is not broken after `ovs` is restarted. Old `vethXXX` interfaces are picked by `ovs` after the restart. 

##### Additional Information

* Operating system and version: CentOS 7

**Comment by danwinship** [comment]
Restarting OVS should cause the SDN pod to restart, and it should reattach the pods then. Is that not happening?

**Comment by metallica127** [comment]
> Restarting OVS should cause the SDN pod to restart, and it should reattach the pods then. Is that not happening?

The sdn pod is restarted along with ovs, but pods network is not attached.

**Comment by metallica127** [comment]
@danwinship we've encountered this issue one more time today. Is there any possible fix?

**Comment by danwinship** [comment]
This may be fixed by #58, but it's not clear if/when that's going to be backported to 3.11. If that is the problem, then you could fix it by stopping the SDN pod before you restart the OVS pod, and then restarting it afterward.

**Comment by Conan-Kudo** [comment]
@danwinship There's a backport PR proposed: https://github.com/openshift/origin/pull/24318
